discord_thread_summarizer_using_openai/discord_utils.bal

```diff
@@ -0,0 +1,55 @@
+import ballerina/time;
+import ballerina/http;
+
+const time:Seconds DISCORD_EPOCH = 1420070400000.00;
+
+http:Client discordClient = check new (DISCORD_API_URL);
+
+function getChannelDetails(int channelId) returns Channel|error {
+    string path = string `/channels/${channelId}`;
+    http:Response response = check discordClient->get(path, headers = {
+        "Authorization": botToken
+    });
+    return (check response.getJsonPayload()).cloneWithType();
+}
```

Comment:
```suggestion
function getChannelDetails(int channelId) returns Channel|error {
    return discordClient->get(string `/channels/${channelId}`, headers = {
        "Authorization": botToken
    });
}
```

Reply:
Addressed in https://github.com/ballerina-guides/ai-samples/pull/69/commits/1dbf76e067fd08cc0430182aa02b7c98d154f3ac